### PR TITLE
Chores/testable epics

### PR DESCRIPTION
**Pull request by srhboo**
In this PR:

- Change parameters of epics so that the ajax function is passed in. This is so we can easily test epics by passing in a fake ajax function.

- index epic file passes in Observable.ajax as the ajax function within context of the working app

### CODE REVIEW

File: src/redux/epics/event.epics.js [hunk 1]
```diff
@@ -10,18 +10,19 @@ const API_KEY_PATH = `?token=${EVENTBRITE_API_KEY}`;
 const CATEGORY_QUERY_PATH = '&location.address=toronto&categories=';
 const CATEGORY_METHOD_PATH = 'search/';
 const reCategoryPath = /^\/category\/\d\d\d$/;
-export const returnEventsActionOnLocationChange = (action$) =>
+
+export const returnEventsActionOnLocationChange = (action$, _) =>
```
Comment (epq) [inline src/redux/epics/event.epics.js]: Do you need to pass in this second argument if you're not going to be using it?

File: src/redux/epics/event.epics.js [hunk 2]
```diff
@@ -37,18 +38,18 @@ export const getEventsEpic = (action$) =>
 
 // SECTION: Single event
 const reEventPath = /^\/event\/(\d)+$/;
-export const returnEventActionOnLocationChange = (action$) =>
+export const returnEventActionOnLocationChange = (action$, _) =>
```
Comment (epq) [inline src/redux/epics/event.epics.js]: Same question as above.

File: src/redux/epics/event.epics.js (hunk 1 above)
Comment (srhboo) [inline src/redux/epics/event.epics.js]: @epq it works without it - I can take it out!

**Comment by auredufour** [comment]
👍

**Pull request merged**